cmd/assetsvc/postgresql_utils_test.go

```diff
@@ -52,6 +52,22 @@ func (f *fakePGManager) QueryAllCharts(query string, args ...interface{}) ([]*mo
 	return chartsResponse, nil
 }
 
+func (f *fakePGManager) InvalidateCache() error {
+	return nil
+}
+
+func (f *fakePGManager) InitTables() error {
+	return nil
+}
+
+func (f *fakePGManager) EnsureRepoExists(namespace, name string) (int, error) {
+	return 0, nil
+}
+
+func (f *fakePGManager) GetDB() dbutils.PostgresDB {
```

Comment:
mm, I don't fully understand this, rather than modifying the interface, isn't it better to just add the `DB` property to the test struct? (I am not sure if that's the problem).

It seems weird to me, in the functional code, to have `.DB` and `.GetDB` but it's true that I have seen that approach before.

Reply:
I can't see how I can add the `DB` property to the test struct. The tests use the non-test `postgresAssetManager` which is defined in the assetsvc as:
```
type postgresAssetManager struct {
    dbutils.PostgresAssetManagerIface
}
```
so the actual `DB` is not exposed at all unless it's available as a method on that interface? (This is slightly different from how it's done in the asset-sync service, unfortunately, which wraps the actual struct rather than the interface, which exposes DB, yet I can't update this one to be the same because the existing assetsvc tests rely on wrapping a fake implementing the same interface).